cognite/client/_api/datapoints.py

```diff
@@ -402,25 +411,154 @@ def retrieve_dataframe(
                 >>> from cognite.client import CogniteClient
                 >>> c = CogniteClient()
                 >>> df = c.datapoints.retrieve_dataframe(id=[1,2,3], start="2w-ago", end="now",
-                ...         aggregates=["average"], granularity="1h")
+                ...         aggregates=["average","sum"], granularity="1h")
+
+            Get a pandas dataframe with the index regularly spaced at 1 minute intervals, missing values completed and without the aggregate name in the columns::
+
+                >>> from cognite.client import CogniteClient
+                >>> c = CogniteClient()
+                >>> df = c.datapoints.retrieve_dataframe(id=[1,2,3], start="2w-ago", end="now",
+                ...         aggregates=["interpolation"], granularity="1m", include_aggregate_name=False, complete="fill,dropna")
         """
         pd = utils._auxiliary.local_import("pandas")
-        id_df = pd.DataFrame()
-        external_id_df = pd.DataFrame()
+
         if id is not None:
-            id_df = self.retrieve(
+            id_dpl = self.retrieve(
                 id=id, start=start, end=end, aggregates=aggregates, granularity=granularity, limit=limit
-            ).to_pandas(column_names="id")
+            )
+            id_df = id_dpl.to_pandas(column_names="id")
+        else:
+            id_df = pd.DataFrame()
+            id_dpl = DatapointsList([])
+
         if external_id is not None:
-            external_id_df = self.retrieve(
+            external_id_dpl = self.retrieve(
                 external_id=external_id,
                 start=start,
                 end=end,
                 aggregates=aggregates,
                 granularity=granularity,
                 limit=limit,
-            ).to_pandas()
-        return pd.concat([id_df, external_id_df], axis="columns")
+            )
+            external_id_df = external_id_dpl.to_pandas()
+        else:
+            external_id_df = pd.DataFrame()
+            external_id_dpl = DatapointsList([])
+
+        df = pd.concat([id_df, external_id_df], axis="columns")
+
+        complete = [s.strip() for s in (complete or "").split(",")]
+        if set(complete) - set(["fill", "dropna", ""]):
```

Comment:
This "set" function call can be replaced with a set literal, so `{"fill", "dropna", ""}` instead.

Reply:
fixed